src/util.ts

```diff
@@ -25,18 +25,27 @@ export async function sendEventLogMessage(guild: Guild, originId: string | null,
 		return;
 	}
 
-	const logChannelId = getDB().get('channels.event-logs');
-	if (!logChannelId) {
-		console.log('Missing log channel!');
-		return;
-	}
-
-	const logChannel = await guild.channels.fetch(logChannelId);
-	
+	const logChannel = await getChannelFromSettings(guild, 'channels.event-logs');
 	if (!logChannel || logChannel.type !== ChannelType.GuildText) {
 		console.log('Missing log channel!');
 		return;
 	}
 
 	await logChannel.send({ embeds: [embed] });
 }
+
+export async function getChannelFromSettings(guild: Guild, channelName: string): Promise<Channel | null> {
+	const channelId = getDB().get(channelName);
```

Comment:
We tend to follow the Go naming conventions for initialisms and acronyms, even outside of Go. So this would be `channelID`